variables.tf

```diff
@@ -35,9 +35,9 @@ variable "dockerhub_password" {
 variable "kube_config" {
   type    = string
   default = "~/.kube/config"
+  description = "if you overide the value, export the 'KUBECONFIG' env variable also. Eg.  export KUBECONFIG=~/.kube/config-files/kubeconfig-aws-tools.yml or set it on the command line KUBECONFIG=~/.kube/config-files/kubeconfig-aws-tools.yml terraform apply"
```

Comment:
Why overriding the parameter with an environment variable? What is the use case / workaround?

Reply:
else the ' resource "kubectl_manifest" "cert_manager_issuer"' fails